capsules/src/tmp006.rs

```diff
@@ -144,7 +144,8 @@ impl<'a> TMP006<'a> {
 
     fn enable_interrupts(&self) {
         // setup interrupts from the sensor
-        self.interrupt_pin.enable_input(InputMode::PullUp);
+        // TODO(alevy): do we need to make sure it's a pull up?
```

Comment:
Oh yeah, I'm totally happy with it being in the board `main.rs` file. Most boards that use a TMP006 would have a pull-up and wouldn't need to do anything at all. It just so happens that Firestorm does not.